manifests/j/JFrog/Conan/1.66/JFrog.Conan.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: JFrog.Conan
+PackageVersion: "1.66"
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: Conan Package Manager_is1
+AppsAndFeaturesEntries:
+- ProductCode: Conan Package Manager_is1
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '{code:DefDirRoot}\Conan'

```

```suggestion
  DefaultInstallLocation: '%ProgramFiles%\Conan'
```

Not sure what `{code:DefDirRoot}` is but don't think that's a support path variable

---

Hey. Thanks for noticing. This one is actually interesting. Googling for this variable name doesn't yield much. It started appearing in Conan manifests since version 2.10.0. Somehow, `wingetcreate` that I used for making this package must have taken this variable from the newer manifests. Which is kind of weird and unexpected in my opinion.

I'll fix it later today.

---

I created a new PR: #209383